### PR TITLE
Bounty coding languages

### LINKED ISSUE #779: Add the following coding language and categories to the bounties website
Add the following coding languages to the bounties website. This should be visible when adding a bounty, in the filters in the main homepage (https://community.sphinx.chat/tickets), and also when editing a bounty:

- Ruby
- Python
- Postgres
- Elastic search

coding lang

These are for frameworks / libraries, Ruby on Rails, React for stakwork

Also add "Design" to category section.
category

### PR DESCRIPTION
**Pull request by cosmicpotato137**
## Describe your changes
Updated language lists in languageLabelStyle.ts and bounties.ts. Still need to add styles list, but I don't know how that manifests.

## Issue ticket number and link
Closes #779 

## Type of change
New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

### CODE REVIEW

**Comment by cosmicpotato137** [comment]
@kevkevinpal 

The coding languages sphinx supports are maintained in like 4 different places. It might be worth refactoring things to have all bounty search and creation options in a single file. Happy to take this on either as another bounty or another commit!

**Comment by ecurrencyhodler** [comment]
@kevkevinpal if you think that's a good idea I can create a bounty for it.

**Comment by cosmicpotato137** [comment]
@kevkevinpal I just made another update adding language tags. It might be worth increasing this bounty a bit but I'm also happy to handle the refactoring on a separate bounty if necessary.

File: sphinx-tribes.exe
Comment (kevkevinpal) [inline sphinx-tribes.exe:1]: not sure why this is commited

File: frontend/app/yarn.lock
Comment (kevkevinpal) [inline frontend/app/yarn.lock:1]: not sure why new yarn.lock if no new packages are added

**Comment by kevkevinpal** [comment]
> @kevkevinpal
> 
> The coding languages sphinx supports are maintained in like 4 different places. It might be worth refactoring things to have all bounty search and creation options in a single file. Happy to take this on either as another bounty or another commit!

ya I agree @ecurrencyhodler if you can create a bounty for it that would be good

**Comment by kevkevinpal** [comment]
> @kevkevinpal I just made another update adding language tags. It might be worth increasing this bounty a bit and I'm also happy to handle the refactoring on a separate bounty if necessary.

@ecurrencyhodler if you're able to bump that would be good

**Comment by ecurrencyhodler** [comment]
Okay. I increased it to 2MM. I'll create a separate bounty to refractor.

File: frontend/app/yarn.lock
Comment (cosmicpotato137) [reply frontend/app/yarn.lock:1]: Yarn may have moved things around since I was trying to get prettier working

File: sphinx-tribes.exe
Comment (cosmicpotato137) [reply sphinx-tribes.exe:1]: I'm not sure either, that's my bad. Should I add .exe files to the gitignore?

File: frontend/app/yarn.lock
Comment (kevkevinpal) [reply frontend/app/yarn.lock:1]: ahh ok we can leave this for now

File: sphinx-tribes.exe
Comment (kevkevinpal) [reply sphinx-tribes.exe:1]: ya if you can add that to the ignore and remove this from the commit that would be great and we can merge this

**Pull request merged**